testing/scenario_app/lib/src/locale_initialization.dart

```diff
@@ -16,6 +16,8 @@ class LocaleInitialization extends Scenario {
       : assert(window != null),
         super(window);
 
+  int tapCount = 0;
```

Comment:
nit, here and elsewhere
```suggestion
  int _tapCount = 0;
```